benchmarks/flatten.py

```diff
@@ -5,9 +5,10 @@
 import arkouda as ak
 
 
-def time_flatten(N, trials):
+def time_flatten(size, trials):
```

Comment:
Nit, but the other benchmarks call this `N_per_locale`

Reply:
oooh yeah i was looking at `Str-in1d`. But it does look like most of the others do `N_per_locale`